### DISCUSSION TITLE
Migrating from Firefly 4.7.9 on Sandstorm to 5.7.10 on Docker

### DISCUSSION THREAD

**Discussion by qirtaiba** [firefly-iii/firefly-iii#6416] [Firefly III]
Hi,

I am running the most recent (but still ancient) version of Firefly that exists for Sandstorm, which is 4.7.9. I want to move this data into a new Firefly 5.7.10 on Docker.

What is the best way of doing this - should I attempt to move the database, or is using the CSV importer the way to go? If so, how much metadata can I expect to lose in the process?

Thanks.

**Comment by JC5** [comment]
Attempt to move the database, then install 5.7.10 on top of it. It *should* migrate flawlessly.

**Comment by qirtaiba** [comment]
Thanks. I tried the CSV import first, and that definitely worked... not flawlessly. So I'm trying the database now. Only problem is... I'm not sure how to access the MySQL that is running inside Sandstorm. Do you know of any documentation on that?

Thanks again.

**Reply by JC5** [reply]
I don't, sorry. Sandstorm has been a while.
